modules/administration-guide/pages/creating-a-telemetry-plugin.adoc

```diff
@@ -377,15 +377,15 @@ components:
 
 
 .Verification steps
-. Verify that the `telemetry-plug-in` container is running in the {devworkspace} pod. Here, this is verified by checking the Workspace view within the editor.
+. Verify that the `telemetry-plugin` container is running in the {devworkspace} pod. Here, this is verified by checking the Workspace view within the editor.
```

Comment:
Is the container name an arbitrary name or as specified by the devs?

Reply:
yes